.cico/cico_openshift_e2e.sh

```diff
@@ -43,9 +55,24 @@ if ! hash operator-sdk 2>/dev/null; then
         rm operator-sdk-${OPERATOR_SDK_VERSION}-x86_64-linux-gnu
 fi
 
+# Function to get all logs and events from devworkspace operator deployments
+function getCheClusterLogs() {
```

Comment:
CheCluster term usage here seems to be copy/paste mistake

Reply:
:O Right